R/draft.R

```diff
@@ -141,13 +141,13 @@ list_template_dirs <- function() {
 
     # check to see if the package includes a template folder
     template_folder <- system.file("rmarkdown", "templates", package = pkg)
-    if (dir_exists(template_folder)) {
+    if (fs::dir_exists(template_folder)) {
```

Comment:
Currently **fs** is only a `Suggests` dependency, and I don't want to make it a hard dependency. If you don't have a strong reason about why `fs::dir_exists` has to be used here, I'd not change it here and [I also hope that this kind of changes can be left out](https://yihui.org/en/2018/02/bite-sized-pull-requests/) because they are not essential.